server/bin/generate-open-api.sh

```diff
@@ -3,7 +3,7 @@
 function mobile {
   rm -rf ../mobile/openapi
   cd ./openapi-generator/templates/mobile/serialization/native
-  wget -O native_class.mustache https://raw.githubusercontent.com/OpenAPITools/openapi-generator/master/modules/openapi-generator/src/main/resources/dart2/serialization/native/native_class.mustache
+  wget -O native_class.mustache https://raw.githubusercontent.com/OpenAPITools/openapi-generator/v6.6.0/modules/openapi-generator/src/main/resources/dart2/serialization/native/native_class.mustache
```

Comment:
It would probably be good to make this a variable in the script to make sure they're all kept in sync.